### PR TITLE
Vue3

### PR DESCRIPTION
**Pull request by jlangy**
### Please check if the PR fulfills these requirements:

- [ ] After these changes, the app was run and still works as expected
- [x] Tests for these changes were added (if applicable)
- [ ] All existing unit tests were run and still pass

### Please specify the type of change this PR introduces (Bug fix, feature addition, content update, chore, etc.):
chore
### Additional Notes:
I had these from before adding the linting rules which is why the format style is old. Might make sense to merge in and then do a single reformatting pull request on all the files. Covers:
- ContinueBar
- CurrencyInput
- ContentModal

### CODE REVIEW

File: src/components/CurrencyInput.vue [hunk 1]
```diff
@@ -130,13 +130,14 @@ export default {
       }
     }
   },
+  emits: ['update:modelValue', 'input', 'blur'],
```
Comment (harrymaynard-maximus) [inline src/components/CurrencyInput.vue:133]: Please break these down with each on its own line.

**Review by harrymaynard-maximus — CHANGES_REQUESTED** [review]

File: src/components/CurrencyInput.vue (hunk 1 above)
Comment (jlangy) [reply src/components/CurrencyInput.vue:133]: Sure, I just left a comment that I had these changes from before the linting rules, didn't run the linter on them to keep the diff small, I was thinking we could bring this in and then run the linter on all files in one pull request without logic changes to make it easier. But could just lint too in this pr if preferable

Comment (jlangy) [reply src/components/CurrencyInput.vue]: I added that here though, the last commit is just linting rules, the component commits show the logic changes alone

**Review by harrymaynard-maximus — APPROVED** [review]
Looks good. I'm kind of on the fence with some of the new linting rules. I guess there's places where it looks nice, and others where it doesn't. We can maybe debate this later.

**Comment by jlangy** [comment]
I agree, probably will want to adjust. Would be nice to get team agreements for a good set in vue projects that we can all agree on and just use the same .eslintrc file across projects

**Pull request merged**